src/frontends/pytorch/src/op/log_softmax.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_log_softmax(const NodeContext& context) {
+    /*
+    aten::log_softmax(
+        Tensor input,
+        int64 dim,
+        dtype dtype = None
+    )
+    */
+    num_inputs_check(context, 1, 3);
```

Comment:
```suggestion
    num_inputs_check(context, 2, 3);
```
You access `dim` on line 26 without checking if it exist